CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# Version 6.0.0
+Released June 14, 2021
```

Comment:
we should add something to the release notes. like "support for sequelize 6.0" and mention that there's telemetry and say how to disable it

(btw, in general, i only do the publish step after the corresponding commit gets merged. that way the commit SHAs are correct)